db/migrations/20191211-index-cclf-imports.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX idx_cclf_beneficiaries_file_id ON public.cclf_beneficiaries USING btree (file_id);
```

Comment:
Can this file be removed now that the indexes are incorporated in the `gorm` migration?

Reply:
Don't we need it in order to migrate the other AWS environments via Jenkins?

Reply:
I was hoping the `gorm` auto-migrate capability would take care of it for us, for free.  Adding the index to the model should achieve this for us.
Maybe you can deploy this branch to `dev` to see if the auto-migrate functionality adds the indexes automatically.

Reply:
You called it--I was not expecting changes of any kind for an existing column, not even an index.  💯 Script deleted.